src/ResourceManager/Sql/Commands.Sql/Location Capabilities/Model/MaxSizeCapabilityModel.cs

```diff
@@ -21,16 +21,11 @@ public class MaxSizeCapabilityModel
         /// <summary>
         /// Gets or sets the max size limit 
         /// </summary>
-        public long Limit { get; set; }
+        public int? Limit { get; set; }
```

Comment:
Shouldn't this be Long?

Reply:
@akromm I saw in SDK, this type is changed to int?, so I changed this type to int? to match the CapabilityModel in new SDK.

Reply:
I think this is ok.